build/Dockerfile.ci

```diff
@@ -110,16 +122,19 @@ ENV OPERATOR=/usr/local/bin/windows-machine-config-operator \
 # Changes needed for our CI
 
 # install make package
-RUN microdnf -y install make tar.x86_64 gzip.x86_64 wget.x86_64
+RUN microdnf -y install make tar.x86_64 gzip.x86_64 wget
```

Comment:
I think it is safe to remove `x86_64` everywhere here. Not sure why it was added in the first place.